Configuration/StandardSequences/python/DigiToRaw_cff.py

```diff
@@ -31,6 +31,8 @@
 
 # Remove siPixelRawData until we have phase1 pixel digis
 eras.phase1Pixel.toReplaceWith(DigiToRaw, DigiToRaw.copyAndExclude([siPixelRawData])) # FIXME
+# same for phase2 tracker
+eras.phase2_tracker.toReplaceWith(DigiToRaw, DigiToRaw.copyAndExclude([siPixelRawData],[rpcpacker])) # FIXME
```

Comment:
I would think removing `rpcpacker` should be somebody else's job than tracker, but ok, we can use this to get feedback who should do it.

The correct syntax would be
```
DigiToRaw.copyAndExclude([siPixelRawData, rpcpacker])
```
i.e. one 2-element list, not two 1-element lists.